### PR TITLE
You got scanned page...

### PR DESCRIPTION
**Pull request by MrSeccubus**
Add a page to https://csirt.divd.nl for scanners to redirect to.

### CODE REVIEW

File: csirt.divd.nl/you_got_scanned.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Don't Panic**
+
+You have been scanned by non-profit Dutch Institute for Vulnerability Disclosure (DIVD; Twitter: @DIVDnl). We scan the internet to detect systems that potentially contain vulnerabilities so we can inform system owners. All to support our mission:
+
+<center>
+We aim to make the digital world safer by reporting vulnerabilities we find in digital systems to the people who can fix them. We have a global reach, but do it Dutch style: open, honest, collaborative and for free.
+</center>
+
+<br>
+We only test systems if our tests pass the criteria of proportionality and subsidiarity, means that our tests are non-intrusive and as light-weight as possible and that there are no other ways to reacht he same effect.
```
Comment (cookiemonster) [inline csirt.divd.nl/you_got_scanned.md:21]: proportionality and subsidiarity, **which** means

Comment (cookiemonster) [reply csirt.divd.nl/you_got_scanned.md:21]: We only test systems if our tests pass the criteria of proportionality and subsidiarity, which means that our tests are non-intrusive and as light-weight as possible and that there are no other ways to reach the same effect.

File: csirt.divd.nl/you_got_scanned.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Additionally, all our research and our researchers are bound by our [code of conduct](https://www.divd.nl/code/).
+
+DIVD does not operate on an opt-in or opt-out basis, butyou can obviously block the scanners above, altough we highly recommend against it.
```
Comment (cookiemonster) [inline csirt.divd.nl/you_got_scanned.md:25]: DIVD does not operate on an opt-in or opt-out basis, but you can block the scanners above, although we highly recommend against it.

File: csirt.divd.nl/you_got_scanned.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+We currently have the following cases open, so you the scan you observed is likely related to one of them:
```
Comment (cookiemonster) [inline csirt.divd.nl/you_got_scanned.md:27]: We currently have the following cases open, so the scan you observed is likely related to one of them:

File: csirt.divd.nl/you_got_scanned.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+===========
+
+When you landed on this page, it is probably because you whitnessed one of our scanners in you logs. We operate the following scanners:
+{% for s in site.data.scanners -%}
+* {{ s.name }} - {{ s.description }}
+{% endfor %}
+
+**Don't Panic**
+
+You have been scanned by non-profit Dutch Institute for Vulnerability Disclosure (DIVD; Twitter: @DIVDnl). We scan the internet to detect systems that potentially contain vulnerabilities so we can inform system owners. All to support our mission:
```
Comment (cookiemonster) [inline csirt.divd.nl/you_got_scanned.md:14]: You have been scanned by the non-profit Dutch Institute for Vulnerability Disclosure (DIVD; Twitter: @DIVDnl). We scan the internet to detect systems that potentially contain vulnerabilities to inform system owners. All to support our mission:

File: csirt.divd.nl/you_got_scanned.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+<center>
+We aim to make the digital world safer by reporting vulnerabilities we find in digital systems to the people who can fix them. We have a global reach, but do it Dutch style: open, honest, collaborative and for free.
```
Comment (cookiemonster) [inline csirt.divd.nl/you_got_scanned.md:17]: We aim to make the digital world safer by reporting vulnerabilities we find in digital systems to the people who can fix them. We have a global reach, but do it Dutch style: open, honest, collaborative, and for free.

**Review by cookiemonster — CHANGES_REQUESTED** [review]
A few minor typos

**Review by Wietsman — APPROVED** [review]

File: csirt.divd.nl/you_got_scanned.md (hunk 5 above)
Comment (MrSeccubus) [reply csirt.divd.nl/you_got_scanned.md:17]: DIt heb ik copy en past van www.divd.nl gedaan, staat het daar ook niet goed? 
Ik zie het verschil tussen die van jou en van www niet. Maar neem hem over.

**Comment by MrSeccubus** [comment]
Please recheck, @cookiemonster

**Review by cookiemonster — APPROVED** [review]

**Pull request merged**